rad/src/operators/string.rs

```diff
@@ -35,6 +36,103 @@ pub fn parse_json_array(input: &RadonString) -> Result<RadonArray, RadError> {
     item.try_into()
 }
 
+fn magic_map_cleaning(input: BTreeMap<String, RadonTypes>) -> BTreeMap<String, RadonTypes> {
+    let mut map = input;
+    let label = match map.remove("_label") {
+        Some(RadonTypes::String(rad_string)) => rad_string.value(),
+        _ => unreachable!("XML parsing error"),
+    };
+
+    match map.remove("_children") {
+        Some(RadonTypes::Array(rad_array)) => {
+            let children = rad_array.value();
+            let children_len = children.len();
+
+            match (map.is_empty(), children_len) {
+                (true, x) if x == 0 => {
+                    map.insert(label, RadonString::from("".to_string()).into());
+                }
+                (true, x) if x == 1 => {
+                    map.insert(label, children.first().unwrap().clone());
```

Comment:
I think this way you can avoid the clone

```suggestion
                    map.insert(label, children.into_iter().next().unwrap());
```